SpatialGDK/Source/SpatialGDK/Private/SpatialGDKSettings.cpp

```diff
@@ -6,8 +6,9 @@
 #include "GameFramework/Actor.h"
 
 #if WITH_EDITOR
-#include "Modules/ModuleManager.h"
 #include "ISettingsModule.h"
+#include "Modules/ModuleManager.h"
+#include "Settings/LevelEditorPlaySettings.h"
```

Comment:
was this an IWYU? I can't see any LevelEditor references within this file from before and you've made no other changes?

Reply:
will revert this change now